src/main/java/model/LocustDistributionData.java

```diff
@@ -0,0 +1,130 @@
+package model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class LocustDistributionData {
+	
+    @SerializedName("100%")
```

Comment:
Why do we need this class?
If we have a value in the CSV file which is not present here it will not be shown, no?

Reply:
Because it's a good way to manage the data. The json model is based on the csv report data and takes in account all the information that contains. Obviously, if in next releases of locust4j the csv contains additional information of something new, we will need to modify it (but this will the same if we read the ifnormation directly from the csv).

Reply:
Yes, this is what I mean with the previous comment from the other class. This is not ideal, as locust4j is not a stable dependency, and can always change. Can you investigate a better and simpler way to handle this dynamically?

Reply:
Maybe we can discuss this in the stand up.